pkgs/development/libraries/libgda/default.nix

```diff
@@ -47,6 +47,6 @@ assert postgresSupport -> postgresql != null;
     homepage = https://www.gnome-db.org/;
     license = [ licenses.lgpl2 licenses.gpl2 ];
     maintainers = gnome3.maintainers;
-    platforms = platforms.linux;
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
not `platforms.unix`?

Reply:
That includes things like freebsd. I’m ok changing it, but I don't have the means to test it

Reply:
Guess it's fine.